### PR TITLE
DOCSP-21808 Connection string can be copied from Compass

### PR DESCRIPTION
**Pull request by jocelyn-mendez1**
Connection string can be copied from recently used databases. Updated Paste Connection String section to reflect that. 

JIRA: https://jira.mongodb.org/browse/DOCSP-21808?filter=-3

STAGING: https://docs-mongodbcom-staging.corp.mongodb.com/compass/docsworker-xlarge/DOCSP-21808/connect/#paste-connection-string

### CODE REVIEW

File: source/includes/steps-starting-compass-paste-string.yaml [hunk 1]
```diff
@@ -10,6 +10,16 @@ content: |
   or the :manual:`DNS Seedlist Connection Format
   </reference/connection-string/#connections-dns-seedlist>`.
 
+  - To obtain the connection string from a recently used |compass-short| 
+    deployment:
+
+    a. Open |compass-short| and view list of :guilabel:`Recent` connections
```
Comment (jmd-mongo) [inline source/includes/steps-starting-compass-paste-string.yaml:16]: [nit]

I think this reads a bit better if we use the definite article: "...and view the list of...".

**Review by jmd-mongo — APPROVED** [review]
Thanks for the changes, @jocelyn-mendez1! This one LGTM mod a small nit.

Thank you!
Joe

File: source/includes/steps-starting-compass-paste-string.yaml (hunk 1 above)
Comment (jocelyn-mendez1) [reply source/includes/steps-starting-compass-paste-string.yaml]: Done.

**Comment by jocelyn-mendez1** [comment]
Hi @mmarcon, I added the small update regarding being able to copy the connection string from a recent deployment. Let me know of any possible changes. Thanks!

**Pull request closed without merging**